### ISSUE TITLE
submitErrors object containing only Error instance is not considered an error

### ISSUE THREAD

**Issue by ps-aux** [final-form/final-form#179]
### Are you submitting a **bug report** or a **feature request**?

Bug; in my opinion.

### What is the current behavior?
 If returning `Error` or object having `Error` as value from `onSubmit` it is detected as not having error.

### Sandbox Link
https://codesandbox.io/s/93m26zqjrr

### What's your environment?
```
 "react-final-form": "^3.6.5",
  "final-form": "^4.10.0",
```

### Other information

This worked in
```
"react-final-form": "2.2.0",
"final-form": "3.0.1",
```

The culprit seems to be:
https://github.com/final-form/final-form/blob/42516e0f15430d0c894d775bc66b731cecd423aa/src/FinalForm.js#L63

A fix along the lines `if (value instance Error) return true` would be sufficient.

**Comment by bertho-zero** [comment]
I have the same issue.

**Comment by erikras** [comment]
Wow. This is browser-specific. Safari returns the error, but Chrome does not.

**Comment by bertho-zero** [comment]
Thank you!

**Comment by erikras** [comment]
Published fix in [`v4.16.2`](https://github.com/final-form/final-form/releases/tag/v4.16.2).
